modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -95,13 +101,39 @@ pullSecret: '<pull_secret>' <6>
 Do not move the `install-config.yaml` and `agent-config.yaml` configuration files to the `openshift` directory.
 ====
 
-. Complete the steps in the link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm#install-configuration-files["Configuration Files"] section of the _Oracle_ documentation to download {oci-ccm-full} and {oci-csi-full} manifests as an archive file and save the archive file in your `openshift` directory. You need the {oci-ccm} manifests for deploying the {oci-ccm} during cluster installation so that {product-title} can connect to the external {oci} platform. You need the {oci-csi} custom manifests for deploying the {oci-csi} driver during cluster installation so that {product-title} can claim required objects from {oci}.
+. If you used a stack to provision OCI infrastructure resources: Copy and paste the `dynamic_custom_manifest` output of the OCI stack into a file titled `manifest.yaml` and save the file in the `openshift` directory.
 
-. Access the custom manifest files that are provided in the link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm#install-configuration-files["Configuration Files"] section of the _Oracle_ documentation.
+. If you did not use a stack to provision OCI infrastructure resources: Download and prepare custom manifests for Oracle.
```

Comment:
```suggestion
. If you did not use a stack to provision OCI infrastructure resources: Download and prepare custom manifests to create an Agent ISO image.
```
By the way, it should not be a step, but rather a note in the above step.

Reply:
This might be tough to do because this step introduces and nests the substeps below that talk about downloading and manually configuring the custom manifests, and we can't really have steps in a note. If you want me to make that relationship more clear, I could maybe end the sentence with a colon like this:
```suggestion
. If you did not use a stack to provision OCI infrastructure resources: Download and prepare custom manifests for to create an Agent ISO image:
```